code/game/objects/structures/flora.dm

```diff
@@ -94,13 +99,30 @@
 	death()
 	return ..()
 
-/obj/structure/flora/pottedplant/attackby(obj/item/weapon/W, mob/user)
-	if (W.edge)
-		user.visible_message(span("warning", "[user] cuts down the [src]"))
-		death()
-		return 1
+/obj/structure/flora/pottedplant/attackby(obj/item/W, mob/user)

```

Comment:
I think there should be a message here as well, in the same vain as when it comes to looking for something.

`user.visible_message("[user] begins digging around inside of \the [src].", "You begin digging around in \the [src], trying to hide \the [W].")`